src/wp-includes/html-api/class-wp-html-unsupported-exception.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare( strict_types=1 );
```

Comment:
First time I'm seeing this construct! Do we support this in Core?

Reply:
Oh not sure. Habit. We probably don't.
It _is_ a bit confusing, because it applies type constraints to calling functions _from this file_, but not in external calling code which calls the functions _found in this file_.